data/report_type.csv

```diff
@@ -0,0 +1,27 @@
+"name"

```

Comment:
This should probably have a `"value"` column that stores what is currently in the `"name"` field and the `"name"` field should store the types in title case?